.stylelintrc.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
         "/color/",
         {
           ignoreValues: [
+            "/\\w+\\.\\$.+/",
```

Comment:
Dart sass allows us to use `@use` for namespaced variables. The stylelint package we are using to enforce variables for colors doesn't understand sass namespaces. So we have to explicitly allow namespaced variables using this regex.